### PR TITLE
Zerologon report improvements

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Improves ZL PR by making code more consistent and fixing report style a bit: 
image
image

### CODE REVIEW

File: monkey/monkey_island/cc/services/reporting/report.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ class ISSUES_DICT(Enum):
         VSFTPD = 13
         DRUPAL = 14
         ZEROLOGON = 15
-        ZEROLOGON_PASSWORD_RESTORED = 16
+        ZEROLOGON_PASSWORD_RESTORE_FAILED = 16
```
Comment (mssalvatore) [inline monkey/monkey_island/cc/services/reporting/report.py:68]: I'm not sure I understand the purpose of this change. I think makes the code harder to follow. I think the code is cleaner if the logic of the telemetry and view are consistent.

File: monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js [hunk 2]
```diff
@@ -370,16 +370,23 @@ class ReportPageComponent extends AuthComponent {
     if (this.state.report.overview.issues[this.Issue.ZEROLOGON_PASSWORD_RESTORE_FAILED]) {
       zerologonOverview.push(<span>
         <WarningIcon/> Automatic password restoration on a domain controller failed!
-        <Button variant={"link"} href={"#"} target={"_blank"} className={"security-report-link"}>
-        Restore your domain controller's password manually.</Button>
+        <Button variant={"link"}
+                href={"https://www.guardicore.com/infectionmonkey/docs/reference/exploiters/zerologon/"}
+                target={"_blank"}
+                className={"security-report-link"}>
```
Comment (mssalvatore) [inline monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:426]: Do the closing '>' characters go on the same line as shown here, or on the next line as shown on line 952?

File: monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js [hunk 3]
```diff
@@ -258,54 +257,102 @@ class ReportPageComponent extends AuthComponent {
...
+                    <li>Stolen SSH keys are used to exploit other machines.</li>}
+                  {this.state.report.overview.issues[this.Issue.STOLEN_CREDS] &&
+                    <li>Stolen credentials are used to exploit other machines.</li>}
+                  {this.state.report.overview.issues[this.Issue.ELASTIC] &&
+                  <li>Elasticsearch servers are vulnerable to
+                    <Button
+                      variant={"link"}
+                      href="https://www.cvedetails.com/cve/cve-2015-1427"
+                      target={"_blank"}
+                      className={"security-report-link"}>CVE-2015-1427
```
Comment (mssalvatore) [inline monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:270]: This is difficult to parse. Can `CVE-2015-1427` go on its own line?

**Review by mssalvatore — CHANGES_REQUESTED** [review]

File: monkey/monkey_island/cc/services/reporting/report.py (hunk 1 above)
Comment (VakarisZ) [reply monkey/monkey_island/cc/services/reporting/report.py:68]: I've tried to document the purpose of this change in the commit message. This is an `ISSUE_DICT`. `ZEROLOGON_PASSWORD_RESTORED` is not an issue. `ZEROLOGON_PASSWORD_RESTORE_FAILED` is an issue and we display warnings if it's `true`, same as with other issues. In report service we create this enum to specify which issues happened and which issues did not so the front end only needs to check `if true render warning`

Comment (mssalvatore) [reply monkey/monkey_island/cc/services/reporting/report.py:68]: OK, makes sense.

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**